internal/client/client_test.go

```diff
@@ -105,7 +105,7 @@ func TestWrapAuth0ClientInfo(t *testing.T) {
 		{
 			name:     "Default client",
 			given:    *DefaultAuth0ClientInfo,
-			expected: "eyJuYW1lIjoiZ28tYXV0aDAiLCJ2ZXJzaW9uIjoibGF0ZXN0In0=",
+			expected: "", // As the default contains dynamic data (runtime.Version) we just assert the value is set.
```

Comment:
Is there a way to mock the dynamic data?

Reply:
I was chatting with @sergiught about the best way to test this, and one recommendation was to wrap `runtime.Version` in an internal interface and then override that internal interface from the tests.

I think what I'm going to do is instead have the tests just decode the header and assert the data structure is correct.

Reply:
Improved the test by decoding and asserting the structure instead of just validating it exists